bin/src/index.js

```diff
@@ -17,6 +17,7 @@ const command = minimist( process.argv.slice( 2 ), {
 		h: 'help',
 		i: 'input',
 		l: 'legacy',
+		z: 'frozen',
```

Comment:
I don't think quite specific feature should have an alias. Writing the full version will be less confusing.

Reply:
Makes sense 👍